src/index.js

```diff
@@ -1,12 +1,28 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import App from './App';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import './css/styles.css';
+/* eslint-disable-next-line */
+import Calculator from "./Components/Calculator"; /* eslint-disable-next-line */
+import Navbar from "./Components/Navbar"; /* eslint-disable-next-line */
+import Home from "./Components/Home"; /* eslint-disable-next-line */
+import Quote from "./Components/Quote"; /* eslint-disable-next-line */
+import Footer from "./Components/Footer";
```

Comment:
- [ ] Kindly avoid disabling linters here and resolve the issues manually. Linters help us write standard code across teams and maintain neat code.

- [ ] Your app still compiles with errors anyway. Kindly look into your file names. For example your `components` folder name is in lower case but you are trying to import a capitalized version `Components`. Kindly fix these errors and the single quote linter errors.

  You can use this import code:
  ```
  import Calculator from './components/calculator';
  import Navbar from './components/Navbar';
  import Home from './components/Home';
  import Quote from './components/Quote';
  import Footer from './components/Footer';
  ```
  ![image](https://user-images.githubusercontent.com/36542986/223670722-d13b3b1b-9e7d-4cce-99e6-fd679305b31e.png)

Reply:
Hello @TedLivist 
I resolve the issues manually but working fine on locally but on GitHub not pass a the eslint when i see the error inside action was missing extinction. thus when i add the extension,  execute error again so that why i use /* eslint-disable-next-line */  i can find any solution if you have any recommend solution I will can fix again Thank you.

Reply:
@Yemisirach Consider pushing let me see the error from Github. The code I suggested for you gives no error locally. It takes care of import names and single quotes errors.

Reply:
Ok